BubbleSheetGrader.py

```diff
@@ -0,0 +1,23 @@
+import cv2
+
+class BubbleSheetGrader:
+    """Class for reading image with BubbleSheet answer and grading that."""
+    def __init__(self, path):
+        """Load and resize image"""
+        self.image = cv2.imread(path)
+        self.image = cv2.resize(self.image, (780, 540))
```

Comment:
It's just a question, but I would like to know why did you choose this 780×540 size ?

Reply:
Hi. I choose this size base on my sample and no other reason.